components/Registration/Mitra/index.vue

```diff
@@ -13,9 +13,9 @@
             <div class="registration-mitra__form-title">
               Daftar Mitra
             </div>
-            <div v-if="isError" class="registration-mitra__form-error-message">
+            <div v-if="isError || isFullnameError || isCompanyError || isEmailError || isPasswordError || isPasswordConfirmError" class="registration-mitra__form-error-message">
```

Comment:
better you refactor it with move all of it to method i think more clean

Reply:
okay kak, thanks for your suggestion. You can see the changes [here ](https://github.com/jabardigitalservice/dedi-landing-page/pull/76/commits/e0780eec3b0a58df59ea49eb0e1cc1476e2db526)